kubernetes/webhook/renderMutatingWebhook.dhall

```diff
@@ -1,40 +1,45 @@
 let k8s =
-        ../k8s/1.15.dhall sha256:4bd5939adb0a5fc83d76e0d69aa3c5a30bc1a5af8f9df515f44b6fc59a0a4815
+        ../k8s/1.15.dhall sha256:9ed8981915875f3bbe08ad7047d92cd181b6ece140af876beecadb8ed079e10a
       ? ../k8s/1.15.dhall
 
 let cert-manager =
-        ../cert-manager/package.dhall sha256:1249e3f2a27a1b46b5bd726c81f9424735a8e85344aba173705147b21560a384
+        ../cert-manager/package.dhall sha256:1640fd5080778e8e9f565cf5790efe18ba9aa57599248b3c75425825be4ef739
       ? ../cert-manager/package.dhall
 
 let certsPath = "/certs"
 
 let Webhook =
-        ./Type.dhall sha256:c833a7c500b51ebe0c1879967d11b476428a885b276d70e5f8cc3b4da09890ad
+        ./Type.dhall sha256:a71ac5b5469ecd2b6823ef59404fdf2878a24138028baa1697a3deadc8e9be73
       ? ./Type.dhall
 
 let labels =
-        ./labels.dhall sha256:05c731fe37f21baf0f03bdbe393472a33ba7dd791b924c5ecc2696042f27d6fc
+        ./labels.dhall sha256:1cf6971c63fef213125cc10c5ecf5c32df954da0e186943693994121485f67e1
       ? ./labels.dhall
 
 let deployment =
```

Comment:
these upgrades are a complete disaster.  manually this will be impossible due to the number of references.  I would need to limit to only CP projects.  

another option is inserting a compatibility layer if possible or perhaps there is an automated means.  this is beyond me and not something I will have time for.

Reply:
can always pin to a different git commit for compatibility, as you've seen you can import any url